pallets/elections/src/lib.rs

```diff
@@ -255,14 +255,14 @@ pub mod pallet {
         type Error = ElectionError;
         type DataProvider = T::DataProvider;
 
-        // The elections are PoA so only the nodes listed in the Members will be elected as validators.
+        // The elections are PoA so only the nodes listed in the Validators will be elected as validators.
         // We calculate the supports for them for the sake of eras payouts.
         fn elect() -> Result<Supports<T::AccountId>, Self::Error> {
             let voters =
                 Self::DataProvider::electing_voters(None).map_err(Self::Error::DataProvider)?;
-            let members = Pallet::<T>::non_reserved_members()
+            let members = NextEraReservedValidators::<T>::get()
```

Comment:
We can change this to `validators` as well.

Reply:
done